src/pages/my-buses/my-buses.html

```diff
@@ -1,6 +1,6 @@
 <ion-header>
   <ion-navbar color="primary">
-    <button ion-button menuToggle>
+    <button ion-button menuToggle aria-label="opens the menu" tabindex=1>
```

Comment:
Lack of partial templates is, in my opinion, the most overlooked key feature that these frameworks should have.